app/core/api/views.py

```diff
@@ -314,3 +316,17 @@ class SdgViewSet(viewsets.ModelViewSet):
     permission_classes = [IsAuthenticated]
     queryset = Sdg.objects.all()
     serializer_class = SdgSerializer
+
+
+@extend_schema_view(
+    list=extend_schema(description="Return a list of all the affiliations"),
+    create=extend_schema(description="Create a new recurring event"),
```

Comment:
These strings need to be updated also. Sorry I was lazy about selecting all the lines.